src/views/textview/TextSegment.js

```diff
@@ -65,6 +65,14 @@ const PaliSanskritSegment = (inputData, segment) => {
       `;
 };
 
+const SanskritSegment = (inputData, segment) => {
+  return html`
+        ${segment}<br />
```

Comment:
I understand you need to do something with Sanskrit but sometimes segments are not entire units but sentences split into multiple parts. Would it not be better to add a linebreak on dandas?

Besides, this is something that does not belong here with the code for the Stemmer but with the text-view. I think it confuses things a bit.

Reply:
Well there are also many files where no proper punctuation is given at all. Look for example at the Mahabharata/bhagavad-gīta, the arguably most important part of the gretil collection. There is no danda included, nothing. With the current formatting code it just turns into a big soup of segments...

Reply:
Because of that I suggest to break after every segment. Even though it might be ugly in the case of some files, better than having the whole MBh get messed up...

Reply:
okay